addon-test-support/scroll.js

```diff
@@ -0,0 +1,47 @@
+import getElementWithAssert from './-private/get-element-with-assert';
+import wait from 'ember-test-helpers/wait';
+
+
+/*
+  Triggers a paint (and therefore flushes any queued up scroll events).
+
+  @method triggerFlushWithPromise
+  @return {RSVP.Promise}
+  @private
+*/
+function triggerFlushWithPromise(){
+  return wait().then(() => {
```

Comment:
I can go back and test, but in ember-scrollable I couldn't get the scroll events to be sychronous for the life of me, even using the wait helper -- the wait helper is really simply waiting an arbitrary amount of time, the browser decides when to trigger the scroll.

Then I plugged this in and it made unicorns appear. I can investigate again to make sure I wasn't going crazy and report back.